tests/lib/devicetree/api/app.overlay

```diff
@@ -440,6 +440,19 @@
 			interrupt-parent = <&test_cpu_intc>;
 		};
 
+		/* same as `test_intc` but extends a different L1 interrupt.
+		 * Required for testing if interrupts are encoded properly for
+		 * nodes consumming interrupts from different aggregators.
+		 */
+		test_intc2: interrupt-controller@cafebabe {
```

Comment:
interesting address, it works but I would suggest that we simply continue from address above (`bbbbdccc`)

Reply:
fixed